patches/0013-mali-support-building-against-4.15.patch

```diff
@@ -108,7 +108,7 @@ Signed-off-by: John Stultz <john.stultz@linaro.org>
  	MALI_DEBUG_ASSERT_POINTER(tim);
 -	tim->timer.data = (unsigned long)data;
 -	tim->timer.function = (timer_timeout_function_t)callback;
-+	timer_setup(&tim->timer, callback, 0);
++	__setup_timer(&tim->timer, callback, &tim->timer, 0);
```

Comment:
Please juste add the definition of timer_setup for the older kernels.

Reply:
Do you mean to place #ifdef LINUX_VERSION_CODE and use different apis?

Reply:
Or define timer_setup api locally and generate 0015 patch?

Reply:
something like this: 

#if kernel < ver
#define timer_setup setup_timer
#endif